docs/about/release-notes.rst

```diff
@@ -51,6 +51,7 @@ Bugfixes
 
 * Binary arithmetic operations such as ``x + x`` of ``x * x``, i.e., with both operands the same, now handle correlations correctly and result in the correct variances in the output `#2709 <https://github.com/scipp/scipp/pull/2709>`_.
 * Made ``__sizeof__`` and related functions more accurate `#2705 <https://github.com/scipp/scipp/pull/2705>`_.
+* 1-D entries with identical dimensions but different coordinate units are no longer plotted on the same axes `#2728 <https://github.com/scipp/scipp/pull/2728>`_.
```

Comment:
Should be in bugfix section.

Reply:
Isn't that the case?

Reply:
Hmm, Github's diff shows it in the "Documentation" section for me 😕